app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -147,7 +147,10 @@ def create
 
     # Create meeting in database
     logger.debug("Creating meeting #{params[:meetingID]} in database.")
-    meeting = Meeting.find_or_create_with_server(params[:meetingID], server)
+
+    moderator_pwd = params[:moderatorPW] || SecureRandom.hex(6)
```

Comment:
We should probably try to match BBB itself as closely as possible in the random password generation: the relevant code is here: https://github.com/bigbluebutton/bigbluebutton/blob/363c80133895ca68c7825b463f8ca22df2d7f918/bbb-common-web/src/main/java/org/bigbluebutton/api/ParamsProcessorUtil.java#L647 (It generates a random alphanumeric 8-character string for the password using this function: http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/RandomStringUtils.html#randomAlphanumeric-int- )

Reply:
It looks like the `SecureRandom` class has this function builtin, you could so `SecureRandom.alphanumeric(8)`